pom.xml

```diff
@@ -370,6 +380,21 @@
               </excludeProperties>
               <failOnNoGitDirectory>false</failOnNoGitDirectory>
               <failOnUnableToExtractRepoInfo>true</failOnUnableToExtractRepoInfo>
+              <!-- used in validateRevision -->
+              <validationProperties>
+                <validationProperty>
+                  <name>validating project version</name>
+                  <value>${project.version}</value>
+                  <!-- <shouldMatchTo>^[0-9\.]*$</shouldMatchTo> -->
+                  <shouldMatchTo><![CDATA[^.*(?<!-SNAPSHOT)$]]></shouldMatchTo>
```

Comment:
nope, thats our demo profile inside the pom. We should be allowed to change this at any time and this should not break anything since it's only gets activated when the profile is active.